west.yml

```diff
@@ -78,6 +78,16 @@ manifest:
       groups:
          - hal
 
+    - name: aipl
+      revision: e80e3f634a12a0c11431cdc3ef21bb3e9ba5e78c
+      repo-path: alif_image-processing-lib
+      path: modules/lib/aipl
+
+    - name: dave2d_driver
```

Comment:
The driver should be placed in HAL. No need to create one more dependency to a separate repository which is managed by us. I understand external modules once those are not managed by us.

Reply:
Personally I think the driver belongs in the Zephyr repository.